### PR TITLE
Translated into Odia -od issue#25

### PR DESCRIPTION
**Pull request by prasanjit15**
Hii,
I have translated the required files into the Odia language. Please merge it. issue#[25](https://github.com/appwrite/appwrite/issues/25)

### CODE REVIEW

File: app/config/locale/translations/od.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+return [
+    'settings.inspire' => '"ଜ୍ଞାନୀ ହେବାର କଳା ହେଉଛି କ’ଣ ଅଣଦେଖା କରାଯିବ ଜାଣିବାର କଳା |"', // This is the line printed in the homepage and console 'view-source'
+    'settings.locale' => 'od',
+    'settings.direction' => 'ltr',
+
+    // Service - Users
+    'account.emails.team' => '%s ଟିମ',
+    'account.emails.verification.title' => 'ଏକାଉଣ୍ଟ ଭେରିଫିକେସନ',
+    'account.emails.verification.body' => 'en.email.auth.confirm.tpl',
```
Comment (spielers) [inline app/config/locale/translations/od.php:11]: you should change en.email.auth.confirm.tpl =>od.email.auth.confirm.tpl
similar with bellow all replace en. with your local language ISO code 

best,
spielers

**Review by spielers — CHANGES_REQUESTED** [review]

**Comment by spielers** [comment]
Thanks for your contributions, I would like to have some required changes @prasanjit15 

best,
spielers

**Comment by prasanjit15** [comment]
> Thanks for your contributions, I would like to have some required changes @prasanjit15
> 
> best,
> spielers

hmm give me a min will do the required

**Comment by prasanjit15** [comment]
@spielers have done the changes. You can verify it now

**Comment by prasanjit15** [comment]
@spielers will you please add the hacktoberfest-accepted label to the PR?

**Comment by spielers** [comment]
@prasanjit15 Yeah we are on it.

**Comment by prasanjit15** [comment]
@spielers will this be merged?

**Comment by eldadfux** [comment]
@prasanjit15 yes i'll merge this, but please update this 2 files as well:

**Configuration and Loading**
Add the new language to Appwrite config file: /app/config/locale/codes.php (order by ABC)
Load the new translation here: ./app/init.php (line: 224) (order by ABC)

**Comment by spielers** [comment]
> @prasanjit15 yes i'll merge this, but please update this 2 files as well:
> 
> **Configuration and Loading**
> Add the new language to Appwrite config file: /app/config/locale/codes.php (order by ABC)
> Load the new translation here: ./app/init.php (line: 224) (order by ABC)

@prasanjit15  Follow the above Instructions Will get you merged!
best,
spielers

**Comment by prasanjit15** [comment]
@eldadfux  @spielers ok will do it

**Comment by prasanjit15** [comment]
> > @prasanjit15 yes i'll merge this, but please update this 2 files as well:
> > **Configuration and Loading**
> > Add the new language to Appwrite config file: /app/config/locale/codes.php (order by ABC)
> > Load the new translation here: ./app/init.php (line: 224) (order by ABC)
> 
> @prasanjit15 Follow the above Instructions Will get you merged!
> best,
> spielers

@spielers Kindly check PR [#673](https://github.com/appwrite/appwrite/pull/673) for the other changes @eldadfux  asked me to do. That is in continuation of this one.

**Comment by spielers** [comment]
Hey, @prasanjit15 we should close This PR now #673  Got merged.

**Comment by prasanjit15** [comment]
Ok sure

**Pull request closed without merging**